static/js/clients.js

```diff
@@ -0,0 +1,143 @@
+            });
+        }
+
+        enableFilters(filterId) {
+            let platformFilterMenu = document.getElementById(filterId+"-menu");
+            for(var platformOption of platformFilterMenu.children) {
+                if(!platformOption.classList.contains("reset-links")) {
+                    let optionId = platformOption.children[0].id;
+                    let checkbox = platformOption.children[0];
+                    if(this.isAndFilter) {
+                        if(checkbox.checked) {
+                            this.allOf.push(platformOption.children[0].id);
+                        }
+                    } else {
+                        if(checkbox.checked) {
+                            this.anyOf.push(platformOption.children[0].id);
+                        }
+                    }
+                    checkbox.addEventListener("change", (event) => {
+                        if(checkbox.checked) {
+                            this.boxWasChecked(optionId);
+                        } else {
+                            this.boxWasUnchecked(optionId);
+                        }
+                    });
+                } else {
+                    platformOption.children[0].addEventListener("mouseup", (event) => {
+                        this.checkAllBoxes(filterId);
+                    });
+                    platformOption.children[1].addEventListener("mouseup", (event) => {
+                        this.uncheckAllBoxes(filterId);
+                    });
+                }
+            }
+        }
+
+        boxWasChecked(optionId) {
+            if(this.isAndFilter) {
+                let filterPos = this.allOf.indexOf(optionId);
+                if(filterPos == -1) {
+                    this.allOf.push(optionId);
+                }
+            } else {
+                let filterPos = this.anyOf.indexOf(optionId);
+                if(filterPos == -1) {
+                    this.anyOf.push(optionId);
+                }
+            }
+            this.refreshCardsView();
+        }
+
+        boxWasUnchecked(optionId) {
+            if(this.isAndFilter) {
+                let filterPos = this.allOf.indexOf(optionId);
+                if(filterPos != -1) {
+                    this.allOf.splice(filterPos, 1);
+                }
+            } else {
+                let filterPos = this.anyOf.indexOf(optionId);
+                if(filterPos != -1) {
+                    this.anyOf.splice(filterPos, 1);
+                }
+            }
+            this.refreshCardsView();
+        }
+
+        checkAllBoxes(filterId) {
+            let platformFilterMenu = document.getElementById(filterId+"-menu");
+            for(let platformOption of platformFilterMenu.children) {
+                if(!platformOption.classList.contains("reset-links")) {
+                        platformOption.children[0].checked = true;
+                        this.boxWasChecked(platformOption.children[0].id);
+                }
```

Comment:
Are you using a formatter at all, actually? 😄

Reply:
No, any recommendation :p ?

Reply:
The one you get by default in VSCode is decent, no idea if that's a thing you can set up independently though.